server/app/jobs/grid_scheduler_job.rb

```diff
@@ -17,7 +17,8 @@ def reschedule_grid(grid)
 
   def perform
     info 'starting to watch services'
-    every(20) do
+    loop do
+      sleep 20
```

Comment:
Right, so this effectively slows down the rescheduler loop if it takes a long time, instead of running it in a busy-loop if the scheduling starts to take more than 20s.

Reply:
Yes.